Assets/Scripts/Utilities/Logger.cs

```diff
@@ -30,7 +30,7 @@ public static void LogException(Exception exception)
         }
     }
 
-    public static void LogAssertion(string message)
+    public static void LogAssertion(object message)
```

Comment:
I respectfully disagree. All these methods, their signatures and parameter names, are made so they exactly match the existing Unity `Debug.Log`. Our custom logger will probably do more in the future, but right now I think it's really important that it do exactly this.